package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hierplane",
-  "version": "0.0.10",
+  "version": "0.0.11",
```

Comment:
Why did this change?

Reply:
actually it shouldn't have. this was a mistake. thanks for reminding me... i need to revert this

Reply:
done